packages/Webkul/Product/src/Type/Configurable.php

```diff
@@ -502,18 +502,70 @@ public function canBeMovedFromWishlistToCart($item)
     }
 
     /**
-     * Get product prices.
-     *
+     * Return variants of configurable product.
      * @return array
-     */
+     * 
+    */
+    public function getAllowedProducts()
+    {
+        static $variants = [];
+
+        if (count($variants)) {
+            return $variants;
+        }
+
+        $variantCollection = $this->product->variants()
+            ->with([
+                'parent',
+                'attribute_values',
+                'price_indices',
+                'inventory_indices',
+                'images',
+                'videos',
+            ])
+            ->get();
+
+        foreach ($variantCollection as $variant) {
+            if ($variant->isSaleable()) {
+                $variants[] = $variant;
+            }
+        }
+
+        return $variants;
+    }
+
+    /**
+     * Return minimum price of variants.
+     * @return array
+    */
+    protected function getVariantPrice()
+    {
+        $prices = [];
+
+        foreach ($this->getAllowedProducts($this->product) as $variant) {
+            $prices[$variant->id] = $variant->getTypeInstance()->getProductPrices();           
```

Comment:
No need to add $prices[$variant->id] as your end result is just taking minimum from the array.

Just pass it like this,

~~~php
$prices[] = $variant->getTypeInstance()->getProductPrices();
~~~